src/test/admin_socket_output.cc

```diff
@@ -14,7 +14,11 @@
 
 #include <iostream>
 #include <regex>                 // For regex, regex_search
-#include <experimental/filesystem> // For extension
+#if __cplusplus >= 201703L && __has_include(<filesystem>)  // For extension
```

Comment:
What's the `__cplusplus >= 201703L` portion doing? Throughout Ceph we normally just have the following (and it would be great to remain consistent):

```
#if __has_include(<filesystem>)
#include <filesystem>
namespace fs = std::filesystem;
#else
#include <experimental/filesystem>
namespace fs = std::experimental::filesystem;
#endif
```

Reply:
It checks the c++ standard version but indeed, it does look unnecessary. I think I can drop it, thanks!